web/views/bundles_view.ex

```diff
@@ -35,6 +36,18 @@ defmodule Cog.V1.BundlesView do
 
   ########################################################################
 
+  defp ordered_compatible_versions(versions) do
```

Comment:
I agree, but I think they're ok for now. Aside from the ordering bit they're just used to format the data for presentation. We should be returning versions from the repository grouped into compatible and incompatible, and sorted. But because of the way we get bundles, that would take a bit of work that I didn't want to bite off since this PR was already getting pretty big.

I would like to do some refactoring on the bundles repository. It's already pretty huge and I'm pretty sure we can consolidate a good bit of the code in there.